_include/q/quickstart-scala.sh

```diff
@@ -18,13 +18,16 @@
 # limitations under the License.
 ################################################################################
 
+source "$(dirname "$0")"/_utils.sh
 
 PACKAGE=quickstart
 
+defaultFlinkVersion="$(extract_parameter 'FlinkStableVersion')"
+
 mvn archetype:generate								\
   -DarchetypeGroupId=org.apache.flink				\
   -DarchetypeArtifactId=flink-quickstart-scala		\
-  -DarchetypeVersion=${1:-1.17.0}							\
```

Comment:
I don't know why it didn't occur to me before but `./quickstart-scala.sh` is failing since 1.17. This is because we removed scala support in 1.17. I created FLINK-33383 to cover this issue.